interface/wx/dataview.h

```diff
@@ -3826,6 +3847,7 @@ class wxDataViewEvent : public wxNotifyEvent
     void SetDataBuffer( void* buf );
     int GetDragFlags() const;
     void SetDropEffect( wxDragResult effect );
+    void SetEditCancelled();
```

Comment:
Does this really need to be in the interface file? For me it's an implementation detail and I'd like to be able to change it in the future, if needed, without caring about compatibility.

Reply:
I added this one in case the programmer needs to create their own instance of the event object for some reason. Without this method there isn't any way for them to set that member. I expect that the actual need to do this is very tiny however, so I'll go ahead and remove it for now.

Reply:
This is again something that we don't do consistently, but I really don't think we should necessarily provide a way for the applications to create arbitrary events anyhow. Creating and sending events that are supposed to be generated by wx controls is often a bug and, in the remaining cases, a hack, so I'd rather avoid advertising the possibility to do this...